### DISCUSSION TITLE
Awesome!

### DISCUSSION THREAD

**Discussion by Ginhuey** [SkyrilHD/Dell-Exx50-Hackintosh#4] [Q&A]
Thanks for sharing this! I've gotten much further with this than the opencore tutorial. This EFI has gotten me to the apple logo where its rebooted several times, always back to the apple logo. Any ideas? Im running a latitude E5550 i5-5200u with intel HD5500.

**Comment by SkyrilHD** [comment]
Make sure you set your BIOS settings correctly. If it still reboots, add `-v debug=0x100 keepsyms=1` as boot-arg in the config, so you can boot in Verbose mode which will give a reason why it does not boot.

**Reply by Ginhuey** [reply]
Thanks for replying! Boot were correct but i went ahead and set default and did it over. Added the bootargs and got verbose with only a few lines before it stopped. Ive reset the nvram as well. 
![9891D143-2AD8-4230-B72C-377F01F4FB60](https://user-images.githubusercontent.com/64372671/111924813-44cc2100-8a74-11eb-994f-bf284f9f2116.jpeg)

**Reply by SkyrilHD** [reply]
Hmm it doesn't even boot to macOS. Remake your installer and redownload the EFI.

Also, do not use any Distros but keep as vanilla as possible

**Reply by terrapinrobotics** [reply]
I have a similar issue on the same machine. Did you ever figure this out?
